kernel/poll.c

```diff
@@ -80,8 +80,9 @@ static inline void add_event(sys_dlist_t *events, struct k_poll_event *event,
 	struct k_poll_event *pending;
 
 	pending = (struct k_poll_event *)sys_dlist_peek_tail(events);
-	if (!pending || _is_t1_higher_prio_than_t2(pending->poller->thread,
-						   poller->thread)) {
+	if ((pending == NULL) ||
```

Comment:
Superfluous parens.

Reply:
this case for me increase readability :)

I'm normalizing this for:

```
if ((x == NULL) && (y != MAX_INT)) {...}
if (x != NULL) {...}
if ((x == NULL) && _some_function_returning_boolean_constant()) {...}
```

Add parentheses around the function call is weird and decide to not add parentheses around the first expression will decrease readability and ...
most cases already have parenthesis around